code/game/objects/structures/signs.dm

```diff
@@ -261,12 +261,12 @@
 
 /obj/structure/sign/safety/hazard
 	name = "hazard semiotic"
-	desc = "Semiotic Standard denoting the nearby presence of a hazard. Watch out."
+	desc = "Semiotic Standard denoting the nearby presence of a hazard. Watch out!"
 	icon_state = "hazard"
 
 /obj/structure/sign/safety/high_rad
 	name = "high radioactivity semiotic"
-	desc = "Semiotic Standard denoting the nearby presence of an area of high radioactivity."
+	desc = "Semiotic Standard denoting the nearby presence of a high radioactivity area."
```

Comment:
this one actually makes more sense as the original (but my suggestion is best)
```suggestion
	desc = "Semiotic Standard denoting the nearby presence of a highly radioactive area."
```

Reply:
Yeah that sounds better, I agree.